extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleJobStore.java

```diff
@@ -36,6 +36,11 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import org.datatransferproject.spi.cloud.storage.JobStore;
```

Comment:
if it's not too much a pain can you revert these files that only changed from the old serializable stuff?

i think you can just git checkout master them as one way to easily revert